fix: bug bash session 2

abanuelo opened the pull request:
### Description

<!-- Provide a brief description of the changes introduced by this pull request -->
1. Clarify Stripe Fee in Form Template Modal
![Screenshot 2025-02-25 at 12 20 56 AM](https://github.com/user-attachments/assets/d4f3d3ca-2ade-4744-9ff8-42a9accbe07c)
2. Reload on Safari works by allowing Auth0 to handle refresh tokens. By default, most [browsers are discontinuing 3rd party cookie support](https://github.com/auth0/auth0-spa-js/blob/main/FAQ.md#why-is-the-user-logged-out-when-they-refresh-the-page-in-their-spa) (and Chrome will soon), so we need to adopt new login style using refresh tokens.  Fixes for that are here and reload is working as expected
![Recording 2025-02-25 at 00 21 44](https://github.com/user-attachments/assets/1d7956d6-c569-4540-8032-3d4e0fe6da9a)
3. Thank you redirection was missing from forms with payment blocks. Should redirect successfully now!
4. Amount was being displayed in cents on the form response modal for an individual response. Should be fixed
5. Fixing 30 character limit for emails in `user.js` model to 100 character limit
6. Adding working photo upload to Team view
![Screenshot 2025-02-25 at 12 39 37 AM](https://github.com/user-attachments/assets/31b9ca47-97d3-41ea-867b-80347389efc0)
7. Clarifying the workflow for "Link to season" option in the create team modal
![Recording 2025-02-25 at 00 46 19](https://github.com/user-attachments/assets/386fe2cd-3204-4850-8945-b7097eb3307c)
8. Adding more space between filter headers and table headers in linear home view
![Screenshot 2025-02-25 at 12 48 13 AM](https://github.com/user-attachments/assets/c2df8425-f40a-4678-aa81-0f4e5e61b812)


### Issue Link

<!-- Provide a link to the related issue on GitHub or another issue tracking system (ie Jira) -->

### Checklist

- [ ] I have tested the changes locally by running `npm run test`
- [ ] I have added or updated relevant documentation
- [ ] I have autoformatted the code by running `npm run eslint`
- [ ] I have added test cases (if applicable)

### Additional Notes

saulRM10 commented on client/src/components/Forms/RegistrationTemplateForm/RegistrationTemplateForm.tsx:
```diff
@@ -538,7 +539,10 @@ const FormTemplateForm: React.FC<RegistrationTemplateFormProps> = ({
               />
               <label className={styles.label} htmlFor="fee">
-                Fee
+                Stripe Fee
```
What if we call it 'Processing Fee' ?

saulRM10 commented on client/src/pages/Teams/Teams.tsx:
```diff
@@ -159,7 +159,16 @@ const Teams = () => {
                       }}
                     >
-                      <img src={team.team_logo} />
+                      {team.team_logo && (
```
can we have a default image for a team if a they don't provide one ?

abanuelo commented:
@saulRM10 addressed your feedback and placeholder images have been added and language has been updated to "Processing Fee"
![Uploading Screenshot 2025-02-25 at 9.12.21 AM.png…]()

jipatino98 commented on client/src/components/Forms/TeamsForm/TeamForm.tsx:
```diff
@@ -33,6 +34,25 @@ const TeamForm: React.FC<TeamFormProps> = ({
+    uploadPhoto();
+  }, [fileUrl]);
+
   console.log("seasonsData", seasonsData);
```
nit: Remove this console log

jipatino98 reviewed (commented):
Looks good! I left a comment about a stray console log that should be removed.

abanuelo commented:
Sorry for closing this out before you finished the review @jipatino98 but added the nit fix to #208

garcialuis commented:
> Looks good! I left a comment about a stray console log that should be removed.

Good catch @jipatino98
